openpype/hosts/blender/utility_scripts/make_paths_relative.py

```diff
@@ -17,6 +17,12 @@
     errors = []
     # Resolve path from source filepath with the relative filepath
     for datablock in get_datablocks_with_filepath(relative=False):
+        # skip render result, compositing and generated images
+        if (
+            isinstance(datablock, bpy.types.Images)
+            and datablock.source in ("GENERATED", "VIEWER")
```

Comment:
Testing is faster against `set`

```suggestion
            and datablock.source in {"GENERATED", "VIEWER"}
```